core/templates/dev/head/pages/donate.html

```diff
@@ -76,7 +76,7 @@ <h2 style="color: #fff; margin-top: 10px;">The Oppia Foundation</h2>
         <h2 style="color: #fff; font-size: 20px; margin-bottom: 20px; margin-top: 20px;">
           Your generous gift funds:</h2>
         <div style="margin-bottom: 16px; text-align: center;">
-          <img ng-src="<[donateImgUrl]>">
+          <img ng-src="/assets/images/general/opp_donate_text.svg">
```

Comment:
This has to be modified via JavaScript (see Donate.js), since we copy assets to static folders when serving in production.